src/ButtonStrip/index.js

```diff
@@ -14,7 +14,7 @@ const ButtonStrip = ({ className, children, compact, start, middle, end }) => (
 
 ButtonStrip.propTypes = {
     className: propTypes.string,
-    children: buttonArrayPropType.isRequired,
+    children: buttonArrayPropType,
```

Comment:
React throws an error when `isRequired` is used on this custom validator, claiming it to be `undefined`. I logged out `buttonArrayPropType` and it isn't `undefined`. Not sure what that's about.

Reply:
Thanks.